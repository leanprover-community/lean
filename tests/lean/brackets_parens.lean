variables {α : Type} [has_one α]
#check {(1 : α)}
#check {(1), (2), (3)}
#check ({(∘), (∘)} : set ((ℕ → ℕ) → (ℕ → ℕ) → (ℕ → ℕ)))