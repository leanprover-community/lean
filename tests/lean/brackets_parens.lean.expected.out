{1} : ?M_1
{1, 2, 3} : ?M_1
{function.comp, function.comp} : set ((ℕ → ℕ) → (ℕ → ℕ) → ℕ → ℕ)
